safe/reportv4/extractors/action_notes.py

```diff
@@ -0,0 +1,80 @@
+# coding=utf-8
+import os
+
+from safe.definitionsv4.exposure import exposure_population, exposure_road, \
+    exposure_all
+from safe.utilities.i18n import tr
+from safe.definitionsv4.hazard_classifications import generic_hazard_classes
+
+__copyright__ = "Copyright 2016, The InaSAFE Project"
+__license__ = "GPL version 3"
+__email__ = "info@inasafe.org"
+__revision__ = '$Format:%H$'
+
+
+def action_checklist_extractor(impact_report, component_metadata):
+    """
+    Extracting action checklist of the exposure layer
```

Comment:
Ah I forgot the convention.
Although, I really don't like it when it has to be on the same line as """. It looks ugly.